composer.json

```diff
@@ -10,11 +10,14 @@
 		},
 		"sort-packages": true
 	},
+	"require-dev": {
+		"nextcloud/ocp": "^25.0",
+		"phpunit/phpunit": "^9.5",
+		"psalm/phar": "^5.9"
```

Comment:
```suggestion
		"vimeo/psalm": "^5.9"
```

Reply:
What's the difference and why is `vimeo/psalm` superiour?

Reply:
It's the original source and I think the phar thing is heading to the grave